### ISSUE TITLE
Assume direction of action

### ISSUE THREAD

**Issue by lucypero** [SublimeText/Origami#171]
I don't want to have to worry about specifying a direction when cloning or moving a file, etc. Just put it on the nearest pane. I use just 2 panes 99% of the time.

**Comment by nschuck** [comment]
agreed! the issue is similar to #123 #122 and #88 . Would be great to have that feature.

**Comment by Jeangowhy** [comment]
> I don't want to have to worry about specifying a direction when cloning or moving a file, etc. Just put it on the nearest pane. I use just 2 panes 99% of the time.

Yep, switch between The Last and The Curren pane is convenient op.

**Comment by rchl** [comment]
Note that the "last" pane is not necessarily the same as "nearby" pane.
